client-common/src/key/private_key.rs

```diff
@@ -49,6 +53,23 @@ impl PrivateKeyAction for PrivateKey {
         Ok(signature)
     }
 
+    fn schnorr_sign_aux(&self, tx: &Transaction, aux_payload: &[u8]) -> Result<SchnorrSignature> {
```

Comment:
```suggestion
    fn schnorr_sign_unsafe(&self, tx: &Transaction, aux_payload: &[u8]) -> Result<SchnorrSignature> {
```

As this is not meant to be called anywhere else in the client and is only used in dev-utils, I suggest the name should reflect it's "dev-only"